LetsBora/LetsBora/Login/LoginView.swift

```diff
@@ -0,0 +1,279 @@
+//
+//  LoginView.swift
+//  LetsBora
+//
+//  Created by Mariana Maiko on 21/03/25.
+//
+
+import UIKit
+
+class LoginView: UIView {
+    private let gradientLayer = CAGradientLayer()
+    
+    lazy private var scrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        return scrollView
+    }()
+    
+    lazy private var logoImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "logoImage")
+        imageView.contentMode = .scaleAspectFit
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+    
+    lazy private var tittleLabel: UILabel = {
```

Comment:
Achei um errinho de digitação:

```suggestion
    lazy private var titleLabel: UILabel = {
```